src/ddprof_input.cc

```diff
@@ -56,11 +56,27 @@
 #define X_HLPK(a, b, c, d, e, f, g, h, i)                                      \
   "  -" #c ", --" #b ", (envvar: " #a ")",
 
-// Helpers for expanding the OPT_TABLE here
+namespace {
+std::string api_key_to_dbg_string(const std::string_view value) {
+  if (value.size() != k_size_api_key) {
+    return "invalid";
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏ why not keep last 4 chars whatever the input string ?